src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/VideoFanTakeover.java

```diff
@@ -1,19 +1,24 @@
 package com.wikia.webdriver.pageobjectsfactory.componentobject.ad;
 
+import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.core.WikiaWebDriver;
 import com.wikia.webdriver.common.core.elemnt.Wait;
+import com.wikia.webdriver.common.contentpatterns.AdsContent;
 
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
 
 public class VideoFanTakeover {
-  public static final double IMAGE_ASPECT_RATIO = 2.459;
-  public static final double VIDEO_ASPECT_RATIO = 1.769;
   private static final int VIDEO_LENGTH = 6000;
-  private static final By VIDEO_IFRAME_SELECTOR = By.cssSelector(".video-ima-container iframe");
-  private static By videoContainerSelector = By.cssSelector(".video-ima-container.hidden");
+  private static final String VIDEO_IFRAME_SELECTOR = "#%s .video-ima-container iframe";
+  private static final String MOBILE_VIDEO_SELECTOR = "#%s .video-ima-container video";
+  private static final String VIDEO_CONTAINER_SELECTOR = "#%s .video-ima-container.hidden";
+  private static final String UI_ELEMENT_SELECTOR = "#%s .overVideoLayer";
+  private static final String FANDOM_URL = "http://www.wikia.com/fandom";
+  private static final int PERCENT = 28;
```

Comment:
The name of this constant isn't very descriptive. Think of maybe a little bit longer but more descriptive name, please :)